src/components/UI/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,20 @@
+'use client'
+import { PropsCheckbox } from '@/types/Checkbox'
+
+export default function Checkbox({ label, isChecked, onCheck }: PropsCheckbox) {
```

Comment:
1. It’s advisable to use destructuring to efficiently pass the remaining props to the input element. This approach ensures that any prop not explicitly pulled out is passed down, making the component more flexible and easier to integrate with form libraries or other parent components.
1.1.  Destructure `onChange` directly and provide a default noop function (`() => {}`). This ensures that the component always has a functional `onChange` handler and prevents potential runtime errors.
1.2. Lock Down the `type` prop: Explicitly destructure the `type` prop and set it to 'checkbox' within the component to prevent it from being overridden. 
1.3. I suggest to add two new props for styling: `labelClassNam`e for the label and `inputClassName` for the input 
1.4. M.b., it will be better to add `false` as default `isChecked` value.

`
const Checkbox: React.FC<PropsCheckbox> = ({
    label,
    isChecked = false,
    onChange = () => {},
    labelClassName = '',
    inputClassName = '',
    ...restProps
}) => {......}
`

2. Additionally, consider renaming onCheck to onChange to align with standard naming conventions in React for event handlers, which enhances readability and maintainability.

Reply:
fixed